source/primer/devops/continuous_integration.rst

```diff
@@ -0,0 +1,117 @@
+
+Once triggered, a Azure pipeline will clone the project and feedstock repositories of the given tudat-related project (tudat, tudatpy, tudat-resources, ...), and for the correct branch (``main`` or ``dev``).
+
+First of all, a distinction is made if the build is for tudatpy on the Linux platform. This is because, afterwards, the tudatpy documentation is generated by Readthedocs using the tudatpy conda package for Linux (as discussed :ref:`below<Readthedocs>`).
+In this case, the docstrings of tudatpy are parsed, generating the documentation in the ``docstrings.h`` header file.
+This is done by calling `build_docstrings.py <https://github.com/tudat-team/tudatpy/blob/01c5335728c52fe776cbc01c016d613b564e3ee0/docs/source/build_docstrings.py>`_ in the `recipe/build.sh <https://github.com/tudat-team/tudatpy-feedstock/blob/77507bc6587ef752ffd01f17b466b6b1d7e1eea5/recipe/build.sh>`_ bash script.
+
+Afterwards, whether the docstrings are generated or not, the conda package is built. This is done by running the sequence of commands from the `recipe/build.sh <https://github.com/tudat-team/tudatpy-feedstock/blob/77507bc6587ef752ffd01f17b466b6b1d7e1eea5/recipe/build.sh>`_ bash script: ``cmake``, ``make``, and ``make install``.
+This is a somewhat similar process as described for users compiling tudatpy locally from the `tudat-bundle repository <https://github.com/tudat-team/tudat-bundle>`_.
+
+Once a build is made, it is then packaged using conda. Among other things, all the required dependencies to install the package are specified, as well as the package metadata. 
+
+Then, as soon as an Azure build and conda package is successfully completed for any platform, the generated conda package is uploaded to Anaconda (as discussed :ref:`below<Anaconda>`).
+
+Finally, logs of the Azure pipeline can be inspected at the following pages: `tudat Azure logs <https://dev.azure.com/tudat-team/feedstock-builds/_build?definitionId=2>`_ / `tudatpy Azure logs <https://dev.azure.com/tudat-team/feedstock-builds/_build?definitionId=3>`_.
+
+Anaconda
+---------
+.. image:: https://anaconda.org/tudat-team/tudat/badges/version.svg
+    :target: https://anaconda.org/tudat-team/tudat
+.. image:: https://anaconda.org/tudat-team/tudatpy/badges/version.svg
+    :target: https://anaconda.org/tudat-team/tudatpy
+
+Anaconda is used to store packages that have been build by Azure. Then, any user can automatically install the packages for its given platform (Linux, Windows, OSx, or OSx arm64), using the ``conda install`` command.
+This will also take care of letting user update packages to the latest version, or to rollback to a specific version.
+
+Readthedocs
+-----------
+.. image:: https://readthedocs.org/projects/tudatpy/badge/?version=latest
+    :target: https://readthedocs.org/projects/tudatpy/builds/
+
+After a full pipeline is completed for all platforms for tudatpy, a trigger is made so the documentation is generated by Readthedocs.
```

Comment:
Where/how exactly is this trigger?

Reply:
This is done trough [these settings in Azure](https://dev.azure.com/tudat-team/feedstock-builds/_apps/hub/ms.vss-ciworkflow.build-ci-hub?_a=edit-build-definition&id=8&view=Tab_Triggers). In the Triggers > Build completion. A trigger is set so that the `tudatpy-readthedocs-trigger` pipeline is run when a `tudatpy-feedstock` pipeline is fully and successfully build. These infos have been added to the docs with 10ac841a1171796eb4ef54a53ac567965ee7634e.